src/home_robot/home_robot/manipulation/grasping.py

```diff
@@ -58,6 +58,10 @@ def plan_to_grasp(
             print("-> could not solve for grasp")
             return None
 
+        # Overwrite standoff pos.z with a really high value so it comes in from above
+        standoff_pos = grasp_pos + np.array([0.0, 0.0, 0.6])
```

Comment:
why is this different and hard-coded?

Reply:
To be able to pick up things inside drawer. Refer my comment about increasing max-height elsewhere. Robot hits top edge of the drawer if high-point of grasp-arc is not moved up higher.